solutions/python-igraph/visualize_cliques/visualize_cliques.py

```diff
@@ -0,0 +1,39 @@
+from igraph import Graph, plot
+import matplotlib.pyplot as plt
+
+if __name__ == "__main__":
+    # Set up the base Zachary Karate Club graph
+    base = Graph.Famous("Zachary")
+    base.vs["name"] = [i + 1 for i in range(base.vcount())]
+    base.vs["label"] = base.vs["name"]
+    base_layout = base.layout("kk")
```

Comment:
The reason why we need to compute a layout separately is so that the graph looks the same in all subplots, correct? This has some bearing on https://github.com/igraph/rigraph/issues/499

Reply:
I just got rid of that line since the default style happens to be "kk".